.github/workflows/check.yml

```diff
@@ -86,4 +86,7 @@ jobs:
       - name: "Check fuzzer with mutation test"
         run: ./scripts/check-fuzzer.sh
 
+      - name: "Check fuzzer reproduction"
+        run: ./scripts/gear.sh test fuzz-repr
```

Comment:
probably need to add to install `cargo nextest`
```yml
- name: "Buid: Init"
  run: ./scripts/gear.sh init cargo
```

Reply:
Right. done here https://github.com/gear-tech/gear/pull/3117/commits/eec074ec7e94b8d2dbdf88d3830c03ece934b35d

I don't run the `gear.sh` script for that, as I don't need the `cargo-hack`.